Sources/ProjectDescription/ResourceSynthesizer.swift

```diff
@@ -38,212 +40,312 @@ public struct ResourceSynthesizer: Codable, Equatable {
         case json
         case yaml
         case files
+
+        public enum Option: Equatable, Codable {
+            case stringValue(String)
+            case intValue(Int)
+            case boolValue(Bool)
+            case doubleValue(Double)
+        }
```

Comment:
Can we make this `ExpressibleByStringLiteral`, `ExpressibleByBooleanLiteral`, etc, so you can just do
```
["separator": "/"]
```
? 🙏

Reply:
That's a really good idea
I'll work on it

Reply:
Done!
- 50fb294932df0d372608d6301d5709b0dae9f715
- 8379cb97c9f6556ece6409a17347adcd79db7c7e